apps/dashr-opiod-endemic/app.R

```diff
@@ -0,0 +1,383 @@
+library(dashR)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+library(purrr)
+library(dplyr)
+library(stringr)
+library(glue)
+library(data.table)
+library(plotly)
+library(readr)
+
+appName <- Sys.getenv("DASH_APP_NAME")
+pathPrefix <- sprintf("/%s/", appName)
+
+Sys.setenv(DASH_ROUTES_PATHNAME_PREFIX = pathPrefix,
+           DASH_REQUESTS_PATHNAME_PREFIX = pathPrefix)
+
+setwd("/app/apps/dashr-opiod-endemic")
+
+app <- Dash$new()
+
+# get lat long data and transform
+df_lat_lon <- as.data.frame(fread("lat_lon_counties.csv"))
+df_lat_lon$FIPS <- str_pad(df_lat_lon$FIPS, 5, pad = "0")
+
+# get lat long data and transform
+df_full_data <- as.data.frame(fread("age_adjusted_death_rate_no_quotes.csv"))
+df_full_data$`County Code` <- str_pad(df_full_data$`County Code`, 5, pad = "0")
+df_full_data$County <- paste0(df_full_data$V2, ", ", df_full_data$County)
+df_full_data <- df_full_data[, -c(1,2)]
+df_full_data$`Age Adjusted Rate` <- replace(df_full_data$`Age Adjusted Rate`, 
+                                            df_full_data$`Age Adjusted Rate` == "Missing", "")
+df_full_data$Deaths <- replace(df_full_data$Deaths, df_full_data$Deaths == "Missing", "")
+df_full_data$`Age Adjusted Rate` <- sub("(Unreliable)", "", df_full_data$`Age Adjusted Rate`, 
+                                       ignore.case = TRUE)
+df_full_data$`Age Adjusted Rate` <- parse_number(as.character(df_full_data$`Age Adjusted Rate`))
+df_full_data$Deaths <- parse_number(as.character(df_full_data$Deaths))
+df_full_data$Deaths <- replace(df_full_data$Deaths , is.na(df_full_data$Deaths), 0)
+df_full_data$`Age Adjusted Rate` <- replace(df_full_data$`Age Adjusted Rate`, 
+                                            is.na(df_full_data$`Age Adjusted Rate`), 0)
+
+YEARS = c(2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012, 2013, 2014, 2015)
+
+BINS <- list("0-2", "2.1-4", "4.1-6", "6.1-8", "8.1-10", "10.1-12", "12.1-14",
+             "14.1-16", "16.1-18", "18.1-20", "20.1-22", "22.1-24", "24.1-26", 
+             "26.1-28", "28.1-30", ">30")
+
+DEFAULT_COLORSCALE <- list("#f2fffb", "#bbffeb", "#98ffe0", "#79ffd6", "#6df0c8", 
+                           "#69e7c0", "#59dab2", "#45d0a5", "#31c194", "#2bb489", 
+                           "#25a27b", "#1e906d", "#188463", "#157658", "#11684d",
+                           "#10523e")
+
+DEFAULT_OPACITY <- 0.8
+
+mapbox_access_token <- "pk.eyJ1IjoieXVua2UwMTIzIiwiYSI6ImNqdm1xc3ZyMjByNm00Nm50MzN1YzNnYncifQ.9RhXTBzJRy1sWeL6tx1ssA"
+mapbox_style <- "mapbox://styles/yunke0123/cjvo2m87d07mk1cpdbo3lvw3j"
```

Comment:
Is this Yunke's personal style from his account? If so, it should probably be changed